content/overview/installation/install-with-docker.md

```diff
@@ -14,13 +14,13 @@ Download the [sample docker-compose file](https://raw.githubusercontent.com/wasm
 
 With the `docker-compose.yml` file in the current directory, start the processes with
 ```
-docker-compose up
+docker compose up
 ```
 
 The host will run until you type ctrl-c or close the terminal window. To start the docker-compose process in the background, add a `-d` flag:
 
 ```
-docker-compose up -d
+docker compose up -d

```

Comment:
I don't know if we should mention this or not, but if you're using linux, you don't get compose v2 for free, I had to manually go install the upgraded docker CLI.

Reply:
If we do mention this,let's link people to https://docs.docker.com/compose/cli-command/#installing-compose-v2